doc/third-party-go-clients.md

```diff
@@ -21,18 +23,19 @@ type Connector interface {
 ```
 
 Its only purpose is to create instances of `api.Connection` that can later be
-used to talk to a given Juju controller with a given set of credentials.  There
-are currently two ways to obtain a `Connector` instance.
+used to talk to a given Juju controller with a given set of credentials. There
+are currently two ways to obtain a `Connector` instance: use the SimpleConnector
+or ClientStoreConnector
 
 ### `SimpleConnector`
 
```

Comment:
This title and next: Phrase like this: 

> Create a `Connector` using ...

It helps give local context and will make it harder for other people to mess the story arc down the line.